### PR TITLE
Document a known issue for IBM Power and Z

### PR DESCRIPTION
**Pull request by alishaIBM**
Jira: https://issues.redhat.com/browse/PROJQUAY-6940

### CODE REVIEW

File: modules/rn_3_11_0.adoc [hunk 1]
```diff
@@ -239,6 +239,11 @@ The {productname} team is aware of the following known issues on the v2 UI:
 
 If you are using the the automatic mode selection, which chooses between light or dark modes depending on the user's browser preference, your operating system appearance is overridden by the browser website appearance setting. If you find that the device-based theme is not working as expect, check your browser appearance setting. This is a known issue and will be fixed in a future version of {productname}. (link:https://issues.redhat.com/browse/PROJQUAY-6903[*PROJQUAY-6903*])
 
+[id="IBM-P-and-Z-known-issue"]
+==== UI Build Log issue for IBM Power & IBM Z
```
Comment (stevsmit) [inline modules/rn_3_11_0.adoc:243]:
```suggestion
=== UI Build Log issue for IBM Power & IBM Z
```

File: modules/rn_3_11_0.adoc [hunk 2]
```diff
@@ -239,6 +239,11 @@ The {productname} team is aware of the following known issues on the v2 UI:
...
+
+UI Build Log issue for IBM Power & IBM Z. UI V2 Build feature is not supported with ODF. It works fine with AWS S3 or any other external object storage. This is a known issue and will be fixed in a future version of {productname}. (link:https://issues.redhat.com/browse/PROJQUAY-6940[*PROJQUAY-6940*]).
```
Comment (stevsmit) [inline modules/rn_3_11_0.adoc:245]:
```suggestion
The UI V2 Build feature is not supported with {odf}. It works with AWS S3 or any other external object storage. This is a known issue and will be fixed in a future version of {productname}. (link:https://issues.redhat.com/browse/PROJQUAY-6940[*PROJQUAY-6940*]).
```

Comment (alishaIBM) [reply modules/rn_3_11_0.adoc]: Ack. Thanks.

File: modules/rn_3_11_0.adoc (hunk 1 above)
Comment (alishaIBM) [reply modules/rn_3_11_0.adoc]: Ack. Thanks.

**Review by yussufsh — APPROVED** [review]
lgtm

**Review by modassarrana89 — APPROVED** [review]

**Comment by alishaIBM** [comment]
@stevsmit request you to merge the PR. Thanks.

**Pull request merged**